src/mesh/primitive.rs

```diff
@@ -85,58 +85,10 @@ pub fn create_cube_smooth(position: [f32; 3], scale: f32) -> Mesh {
 
 #[allow(dead_code)]
 pub fn create_cube_sharp(position: [f32; 3], scale: f32) -> Mesh {
-    #[rustfmt::skip]
-    let vertex_positions = vec![
-        // back
-        v(-1.0,  1.0, -1.0, position, scale),
-        v(-1.0,  1.0,  1.0, position, scale),
-        v( 1.0,  1.0,  1.0, position, scale),
-        v( 1.0,  1.0, -1.0, position, scale),
-        // front
-        v(-1.0, -1.0, -1.0, position, scale),
-        v( 1.0, -1.0, -1.0, position, scale),
-        v( 1.0, -1.0,  1.0, position, scale),
-        v(-1.0, -1.0,  1.0, position, scale),
-    ];
-
-    #[rustfmt::skip]
-    let vertex_normals = vec![
-        // back
-        Vector3::new( 0.0,  1.0,  0.0),
-        // front
-        Vector3::new( 0.0, -1.0,  0.0),
-        // top
-        Vector3::new( 0.0,  0.0,  1.0),
-        // bottom
-        Vector3::new( 0.0,  0.0, -1.0),
-        // right
-        Vector3::new( 1.0,  0.0,  0.0),
-        // left
-        Vector3::new(-1.0,  0.0,  0.0),
-    ];
-
-    let faces = vec![
-        // back
-        TriangleFace::new_separate(0, 1, 2, 0, 0, 0),
-        TriangleFace::new_separate(2, 3, 0, 0, 0, 0),
-        // front
-        TriangleFace::new_separate(4, 5, 6, 1, 1, 1),
-        TriangleFace::new_separate(6, 7, 4, 1, 1, 1),
-        // top
-        TriangleFace::new_separate(7, 6, 1, 2, 2, 2),
-        TriangleFace::new_separate(2, 1, 6, 2, 2, 2),
-        // bottom
-        TriangleFace::new_separate(5, 0, 3, 3, 3, 3),
-        TriangleFace::new_separate(0, 5, 4, 3, 3, 3),
-        // right
-        TriangleFace::new_separate(6, 3, 2, 4, 4, 4),
-        TriangleFace::new_separate(3, 6, 5, 4, 4, 4),
-        // left
-        TriangleFace::new_separate(4, 7, 0, 5, 5, 5),
-        TriangleFace::new_separate(1, 0, 7, 5, 5, 5),
-    ];
-
-    Mesh::from_triangle_faces_with_vertices_and_normals(faces, vertex_positions, vertex_normals)
+    create_box_sharp(
+        Point3::new(position[0], position[1], position[2]),
+        [scale, scale, scale],
+    )
```

Comment:
Why do we need both box and cube? They are the same thing. If it is about the difference in default scale, cube doesn't care. We should have only one of these.

Reply:
> > Yes. Mainly because I wasn't planning to touch any existing code, just needed the box. Nevertheless, the smooth cube is not used anywhere, not even in any tests. Should I remove it completely?
> 
> If you think it will never be useful, than yeah, let's remove it. 

Yes, let's remove it then. I don't feel like computing the normals for a box. I'm about to make a smooth normal calculator for an existing mesh tomorrow morning anyway. I'll also make the normal strategy, yay!

> Btw, is box a more standard name than cube?

Yes, I've never seen this being called a Cube. I think it's because a box is more general and cube is just it's subset.